dyn_em/module_initialize_ideal.F

```diff
@@ -631,6 +631,19 @@ SUBROUTINE init_domain_rk ( grid &
      DO k=1, kde
       grid%znw(k) = model_config_rec%eta_levels(k)
      ENDDO
+            !Check the value of the first and last eta level for our domain,
```

Comment:
@dudhia 
Jimy,
Would you indent this so that it lines up with the rest of the code.

Reply:
done

Reply:
I am OK with this PR now.